CHANGES.txt

```diff
@@ -20,6 +20,7 @@ New checks:
   in Python 3.7. PR #684.
 * Add E252 error for missing whitespace around equal sign in type annotated
   function arguments with defaults values. PR #717.
+* Add E117 error for over-indented code blocks.
```

Comment:
I don't believe this is the right place to add this because 2.4.0 is already released